framework/include/controls/Control.h

```diff
@@ -114,11 +114,11 @@ class Control :
   void setControllableValueByName(const std::string & tag, const std::string & param_name, const T & value, bool warn_when_values_differ = false);
   ///@}
 
-private:
-
   /// A reference to the InputParameterWarehouse which is used for access the parameter objects
   InputParameterWarehouse & _input_parameter_warehouse;
 
+private:
+
   /// Helper method for retrieving controllable parameters
   template<typename T>
   ControllableParameter<T> getControllableParameterHelper(const MooseObjectParameterName & desired, bool warn_when_values_differ, bool mark_as_set = false);
```

Comment:
Okay.  This change doesn't matter much since I can just do:
_another_input_parameter_warehouse(_app.getInputParameterWarehouse())
and create my own version of the variable in a Control subclass.  If you would like a patch with just the InputParameters::getControllableParameters()  change I can do that instead.